docs/TektonChain.md

```diff
@@ -45,6 +45,14 @@ It is recommended to install the component through [TektonConfig](./TektonConfig
 ## Chain Config
 
 There are some fields which you can define on Tekton Chains CR to configure the behaviour of the chains
+If nothing is set, the operator sets these default properties:
+  artifacts.taskrun.format: in-toto
+  artifacts.taskrun.storage: oci
+  artifacts.oci.storage: oci
+  artifacts.oci.format: simplesigning
+  artifacts.pipelinerun.format: in-toto
+  artifacts.pipelinerun.storage: oci
+
```

Comment:
I think we should have this default values which are defined here https://github.com/tektoncd/chains/blob/main/config/100-deployment.yaml#L45-L52

Since we are creating x509 secret keys, does it makes sense to add the default signer as x509 ?